src/mumble/PulseAudio.cpp

```diff
@@ -371,6 +377,18 @@ void PulseAudioSystem::eventCallback(pa_mainloop_api *api, pa_defer_event *) {
 	}
 }
 
+void PulseAudioSystem::corkStream(const bool corked)
+{
+	// TODO: Do I need this lock?
+	if (pasInput)
+	{
+		pa_threaded_mainloop_lock(pam);
+		pa_stream_cork(pasInput, corked, NULL, NULL);
+		pa_threaded_mainloop_unlock(pam);
+	}
+	return;
```

Comment:
This seems very unnecessary to me. Why would you add a return-statement returning void at the end of a function body?

Reply:
Excessive verbosity removed by 2d30973d1b5324dd4255bb54b34d6902ae7bc859.